chromadb/utils/embedding_functions.py

```diff
@@ -244,6 +245,67 @@ def __call__(self, input: Documents) -> Embeddings:
         ]
 
 
+class VoyageAIEmbeddingFunction(EmbeddingFunction[Documents]):
```

Comment:
Sure, great effort and the code looks fine.
There is one thing i'd like to ask to be present in the other PR: the batch_size is not used during calling the embed function - in my PR, there is a loop, iterating over all the input documents with batch_size chunks. This is missing from the other PR.
Can you please add the same to the other PR?

Reply:
Hey @fzowl, I have a comment about this below. I think batching is a good user experience until it isn't. I prefer that we move this whole functionality in an utility function(s) that can be applied to all embedding functions instead of repeating the same code for every EF. Wdyt?